bundles/analysis/analyse/view/DrawControls.js

```diff
@@ -311,8 +311,6 @@ Oskari.clazz.define(
          */
         deactivateSelectTools: function () {
             var me = this;
-
-            me.selectionButtonsRenderer.removeButtonSelection();
```

Comment:
Was this intended? The PR is all about terrain profile, but this change is for analyse functionality?